src/test/java/net/sf/jabref/model/entry/BibEntryTests.java

```diff
@@ -212,16 +212,21 @@ public void setNullField() {
 
     @Test(expected = NullPointerException.class)
     public void addNullKeywordThrowsNPE() {
-        keywordEntry.addKeyword(null);
+        keywordEntry.addKeyword(null, Globals.prefs.get(JabRefPreferences.KEYWORD_SEPARATOR));
```

Comment:
Well, if the KeywordGroup has problems with different separators, then one should write new tests for this separator. But then it would be easier to pass them directly to the method instead of setting a preference. Anyway, logic should be independent on the preferences and so should be their tests.